contracts/PartyBid.sol

```diff
@@ -116,7 +129,7 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
 
     event Bid(uint256 amount);
 
-    event Finalized(PartyStatus result, uint256 totalSpent, uint256 fee, uint256 totalContributed);
+    event Finalized(PartyStatus result, uint256 totalSpent, uint256 ethFee, uint256 totalContributed);
```

Comment:
`ethFee` is a better descriptor but can we leave this the same for the sake of consistency with event pipelines like DuneAnalytics

Reply:
yep fair

Reply:
https://github.com/PartyDAO/partybid/commit/0a0c051109bfddcbe845d6f2a57383806f112e73